src/pages/home/report/ReportFooter.tsx

```diff
@@ -129,13 +129,14 @@ function ReportFooter({
              * Group 2: Optional email group between \s+....\s* start rule with @+valid email or short mention
              * Group 3: Title is remaining characters
              */
-            const taskRegex = /^\[\]\s+(?:@([^\s@]+(?:@\w+\.\w+)?))?\s*([\s\S]*)/;
+            const emailWithOptionalDomainRegex = /(?=((?=[\w'#%+-]+(?:\.[\w'#%+-]+)*@?)[\w\.'#%+-]{1,64}(?:@(?:(?=[a-z\d]+(?:-+[a-z\d]+)*\.)(?:[a-z\d-]{1,63}\.)+[a-z]{2,63}))?(?= |_|\b))(?<end>.*))\S{3,254}(?=\k<end>$)/
```

Comment:
I decided to use the email [regex from `expensify-common`](https://github.com/Expensify/expensify-common/blob/75869597915c75812e85880c686294f12c112698/lib/CONST.ts#L3-L4). The only differences is the optional domain.

<img width="1485" alt="image" src="https://github.com/user-attachments/assets/4a76d4c5-04f5-4385-a2b8-cce7887b36fb">

Reply:
Oh, I just [removed the unnecessary character](https://github.com/Expensify/App/pull/49685/commits/fb32c16bb3c03ce04e5dc17255eb4d4ff103d05f) because the linter asked to.